packages/backend/server/tests/app/selfhost.e2e.ts

```diff
@@ -101,6 +122,67 @@ test('should always return static asset files', async t => {
   t.is(res.text, "const name = 'affine-admin'");
 });
 
+test('doc renderer should return mobile assets for mobile user agent', async t => {
+  let res = await request(t.context.app.getHttpServer())
+    .get(
+      '/workspace/3aa2e665-7b0d-41c6-9979-db17c8d93836/Sabgfj_trVBY6_iM_6uaO'
+    )
+    .set('User-Agent', mobileUAString)
+    .expect(200);
+
+  t.true(res.text.includes('AFFiNE Mobile'));
+
+  res = await request(t.context.app.getHttpServer())
+    .get(
+      '/workspace/3aa2e665-7b0d-41c6-9979-db17c8d93836/Sabgfj_trVBY6_iM_6uaO'
+    )
+    .expect(200);
+
+  t.false(res.text.includes('AFFiNE Mobile'));
```

Comment:
app tests should only contain basic healthy tests
I will add doc renderer tests separately later

Reply:
All right, what can I do for this? And this test case is working on my laptop, but failed in github actions, I will see it when I have time

Reply:
I will append a commit to fix